test/resources/test_container/full/tests/test/docker_environment_test.py

```diff
@@ -15,7 +19,7 @@ def test_connect_from_udf_to_other_container(self):
             self.query(udf.fixindent("CREATE SCHEMA %s" % schema))
             self.query(udf.fixindent("OPEN SCHEMA %s" % schema))
             self.query(udf.fixindent('''
-                CREATE OR REPLACE PYTHON SCALAR SCRIPT connect_container(host varchar(1000), port int)  returns int AS
+                CREATE OR REPLACE PYTHON3 SCALAR SCRIPT connect_container(host varchar(1000), port int)  returns int AS
```

Comment:
I know this is just related but not an issue here in the code, still question regarding the UDFs:
Wouldn't it make sense to have an "alias" `PYTHON3` to `PYTHON` so it will be easier and more 
straight forward to the user. We only support python3 at this point anyway.